public/js/index.js

```diff
@@ -404,15 +404,19 @@ function initRepository() {
         $('.enable-system').change(function () {
             if (this.checked) {
                 $($(this).data('target')).removeClass('disabled');
+                if ($(this).data('context') != undefined) $($(this).data('context')).addClass('disabled');
```

Comment:
Do not use `undefined` correct would be `typeof $(this).data('context') !== 'undefined'` or `!$(this).data('context')`